app.py

```diff
@@ -16,37 +16,28 @@
 
 from dotenv import load_dotenv
 from flask import Flask, render_template, request
+
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
 from ibm_watson import NaturalLanguageClassifierV1
 
 DEBUG = True
 app = Flask(__name__)
 
 load_dotenv(os.path.join(os.path.dirname(__file__), ".env"))
 
-nlc_username = os.environ.get("NATURAL_LANGUAGE_CLASSIFIER_USERNAME")
-nlc_password = os.environ.get("NATURAL_LANGUAGE_CLASSIFIER_PASSWORD")
-nlc_iam_apikey = os.environ.get("NATURAL_LANGUAGE_CLASSIFIER_IAM_APIKEY")
+nlc_iam_apikey = os.environ.get("NLC_IAM_APIKEY")
 classifier_id = os.environ.get("CLASSIFIER_ID")
 
-# Use provided credentials from environment or pull from IBM Cloud VCAP
-if nlc_iam_apikey != "placeholder":
-    NLC_SERVICE = NaturalLanguageClassifierV1(
-      iam_apikey=nlc_iam_apikey
-    )
-elif nlc_username != "placeholder":
-    NLC_SERVICE = NaturalLanguageClassifierV1(
-      username=nlc_username,
-      password=nlc_password
-    )
-else:
-    NLC_SERVICE = NaturalLanguageClassifierV1()
+authenticator = IAMAuthenticator(nlc_iam_apikey)
+NLC_SERVICE = NaturalLanguageClassifierV1(authenticator=authenticator)
```

Comment:
^ will work. Alternatively I think you can also do: `NLC_SERVICE = NaturalLanguageClassifierV1(version=version)`. if you have .env setup correctly it will get picked up.